api/v1beta1/infraenv_types.go

```diff
@@ -60,7 +60,6 @@ type InfraEnvSpec struct {
 	// +optional
 	SSHAuthorizedKey string `json:"sshAuthorizedKey,omitempty"`
 
-	// This field is currently not used. The Pull Secret from ClusterDeployment will be used for the ISO creation.
```

Comment:
if you are removing this part you need to remove the update from cluster deployment controller https://github.com/openshift/assisted-service/blob/8eedf068bcc104c66a1334a94e54598aa507b9ee/internal/controller/controllers/clusterdeployments_controller.go#L716-L720
Suggesting to split it into a different PR.

Reply:
Sure